tests/system/Database/Live/WriteTypeQueryTest.php

```diff
@@ -48,11 +48,49 @@ public function testInsert(): void
 
         $this->assertTrue($this->db->isWriteType($sql));
 
-        if ($this->db->DBDriver === 'Postgre') {
-            $sql = "INSERT INTO my_table (col1, col2) VALUES ('Joe', 'Cool') RETURNING id;";
+        $sql = "WITH seqvals AS (SELECT '3' AS seqval)INSERT INTO my_table (col1, col2) SELECT 'Joe', seqval FROM seqvals;";
```

Comment:
```suggestion
        $sql = "WITH seqvals AS (SELECT '3' AS seqval) INSERT INTO my_table (col1, col2) SELECT 'Joe', seqval FROM seqvals;";
```

Reply:
Hello,

The space was left out deliberately in this test.

Reply:
Okay, then can you show it as a test method name?